docs/ci_runner_setup/linux/README.md

```diff
@@ -0,0 +1,78 @@
+## Linux CI runner setup
+
+This directory contains documentation and scripts about setting up a Linux CI Runner for [`ROCm`](https://github.com/ROCm) organization and used by [`TheRock`](https://github.com/ROCm/TheRock) repository.
+
+Note: you must have sufficient permissions to access [ROCm runner page](https://github.com/organizations/ROCm/settings/actions/runners)
+
+### Setup
+
+For brand new machines that do not that ROCm or Docker installed, please follow these steps. Otherwise, please skip to step 3.
+
+1. Install ROCm to the machine using `sudo ./rocm_install.sh`. This script will install ROCm 6.4 and AMD drivers for Ubuntu24, then it will reboot the system.
+
+   - Rebooting the system is required to load ROCm.
+   - If you have a different Linux distribution, follow [ROCm installation quick start guide](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/quick-start.html)
+   - <b>After reboot, please try `rocminfo` and `rocm-smi` to make sure ROCm is loaded and drivers are installed.</b> If there are issues, please try each command in `rocm_install.sh` instead.
+
+1. If docker is not installed, please run `sudo ./docker_install.sh`. This script will download docker for Ubuntu.
```

Comment:
Do all the commands in the script need `sudo` if you always run the script itself with `sudo`?